core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -282,21 +290,25 @@ class AdminApiServiceSpec extends ApiServiceSpec {
 
     import UserAuthJsonSupport.CreateRawlsBillingProjectFullRequestFormat
 
-    Post(s"/billing", httpJson(createRequest)) ~>
-      sealRoute(services.billingRoutes) ~>
-      check {
-        assertResult(StatusCodes.Created) {
-          status
+    withStatsD {
+      Post(s"/billing", httpJson(createRequest)) ~> services.sealedInstrumentedRoutes ~>
```

Comment:
presumably nothing is lost here by posting this to all the routes combined rather than just the billing routes